src/dropdown/dropdown.ts

```diff
@@ -30,29 +31,69 @@ export class NgbDropdownMenu {
   placement: Placement = 'bottom';
   isOpen = false;
 
+  private _dropdownElement;
+  private _bodyContainer: HTMLElement;
+
   constructor(
       @Inject(forwardRef(() => NgbDropdown)) public dropdown, private _elementRef: ElementRef<HTMLElement>,
-      private _renderer: Renderer2) {}
+      private _renderer: Renderer2) {
+    this._dropdownElement = this._elementRef.nativeElement.parentNode;
+  }
 
   isEventFrom($event) { return this._elementRef.nativeElement.contains($event.target); }
 
-  position(triggerEl, placement) {
-    this.applyPlacement(positionElements(triggerEl, this._elementRef.nativeElement, placement));
+  position(triggerEl, placement, container) {
+    this.applyPlacement(
+        positionElements(
+            triggerEl, this._bodyContainer || this._elementRef.nativeElement, placement, container === 'body'));
+  }
+
+  applyContainer(container: null | 'body' = null) {
+    const nativeElement = this._elementRef.nativeElement;
+    const renderer = this._renderer;
+    const body = document.body;
+
+    if (container === 'body') {
+      this._bodyContainer = this._bodyContainer || document.createElement('div');
+      const bodyContainer = this._bodyContainer;
+
+      // Override some styles to have the positionning working
+      renderer.setStyle(this._bodyContainer, 'position', 'absolute');
+      renderer.setStyle(nativeElement, 'position', 'static');
+
+      renderer.appendChild(bodyContainer, nativeElement);
+      renderer.appendChild(body, bodyContainer);
+    } else {
+      this._dropdownElement.appendChild(nativeElement);
+      renderer.removeStyle(nativeElement, 'position');
+      if (this._bodyContainer) {
+        renderer.removeChild(body, this._bodyContainer);
+        this._bodyContainer = null;
+      }
+    }
   }
 
   applyPlacement(_placement: Placement) {
+    const renderer = this._renderer;
+    const dropdownElement = this._dropdownElement;
+
     // remove the current placement classes
-    this._renderer.removeClass(this._elementRef.nativeElement.parentNode, 'dropup');
-    this._renderer.removeClass(this._elementRef.nativeElement.parentNode, 'dropdown');
+    renderer.removeClass(dropdownElement, 'dropup');
+    renderer.removeClass(dropdownElement, 'dropdown');
     this.placement = _placement;
+
     /**
```

Comment:
Could you please take the opportunity of this PR to correct this comment by using a standard `/*` instead of a documentation one `/**` ?